lib/measures/ImproperlySizedEquipment/measure.rb

```diff
@@ -65,11 +70,11 @@ def run(model, runner, user_arguments)
     # use the built-in error checking
     if !runner.validateUserArguments(arguments(model), user_arguments)
       return false
-    end
+    end  
 
     equip_choice = runner.getStringArgumentValue('equip_choice', user_arguments)
     sizing_ratio = runner.getDoubleArgumentValue('sizing_ratio', user_arguments)
-    hard_size = runner.getBoolArgumentValue('hard_size', user_arguments)
+    hard_size = false#runner.getBoolArgumentValue('hard_size', user_arguments)
```

Comment:
oh I did not know you can call sizing run and applying sizing directly from standards. good to know. so shouldn't this hard_size should be read from input argument so 1) if model is hard sized already (and if this arg is unchecked), you skip this hardsizing process and 2) if model needs hard sizing (and if arg is checked), you go through hard sizing? or am I incorrectly understanding this process

Reply:
Yes it should be reading from the argument. I changed it here because it was causing the measure to crash and I didn't have time to debug the exact reason. Also in OpenStudioApplication (which I was using for testing) check boxes are broken so you can't deselect a boolean argument.